src/site/content/en/blog/pwa-url-handler/index.md

```diff
@@ -0,0 +1,291 @@
+devices. If you want to share your favorite track for your friend to enjoy, you can send them a deep
+link like `https://music.example.com/rick-astley/never-gonna-give-you-up`. Since this link is pretty
+long, the developers of `music.example.com` may have decided to add an additional short link to each
+track, like, for example, `https://🎵.example.com/r-a/n-g-g-y-u`.
+
+PWA as URL Handlers is a proposal that allows apps like `music.example.com` to register themselves
+as URL handlers for URLs that match patterns like `music.example.com`, `*.music.example.com`, or
+`🎵.example.com`, so that links from outside of the PWA, for example, from an instant messenger
+application or an email client, open in the installed PWA rather than in a browser tab.
+
+The PWA as URL Handlers proposal consists of two additions:
+
+1. The `"url_handlers"` Web App Manifest member.
+1. The `web-app-origin-association` file format for validating out-of-scope URL associations.
+
+### Suggested use cases for PWAs as URL Handlers {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Music or video streaming sites so track links or playlist links open in the player experience of
+  the app.
+- News or RSS readers so followed or subscribed-to sites open in the app's reader mode experience.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use PWAs as URL Handlers {: #use }
+
+### Enabling via about://flags
+
+To experiment with PWAs as URL Handlers locally, without an origin trial token, enable the
+`#enable-desktop-pwas-url-handling` flag in `about://flags`.
+
+{% Aside %} Support for the PWAs as URL Handlers feature is limited to desktop operating systems
+like Window, macOS, and Linux. On Android, Chromium browsers install PWAs by generating a
+[WebAPK](https://developers.google.com/web/fundamentals/integration/webapks), which can register a
+set of intent filters for all URLs within the scope of the app. This means that PWAs already handle
+associated URLs on Android at the operating system level using intent filters. {% endAside %}
+
+### The `"url_handlers"` Web App Manifest member
+
+To associate an installed PWA with URL patterns, these patterns need to be specified in the Web App
+Manifest. This happens through the `"url_handlers"` member. It accepts an array of objects with the
+following property:
+
+- `origin`: A required `string`, which is a pattern for matching origins. These patterns are allowed
```

Comment:
Maybe "with an `origin` property, which is a required string that..." If there's only one, you really shouldn't use a bullet with a run-in heading.

Reply:
Done, bullet be gone.